README.md

```diff
@@ -147,3 +147,32 @@ Click the `Allow Anyway` button.
 Building and testing is most easily performed with `make build` and `make test` respectively.
 
 The PGP key used for encrypting the test cases is found in `test/testing-key.pgp`. You can import it with `gpg --import test/testing-key.pgp`.
+
+## Transitioning to Terraform 0.13 (0.12.20+) provider required blocks.
+
+With Terraform 0.13 (0.12.20+), providers are available/downloaded via the [terraform registry](https://registry.terraform.io/providers/carlpett/sops/latest) via a required_providers block (Yay!)
+
+```
+terraform {
+  required_version = ">= 0.13.0"
+  required_providers {
+    sops = {
+      source = "carlpett/sops"
+      version = "0.5.1"
+    }
+  }
+}
+```
+A prerequisite when converting is that you must from remove the data source block from the previous SOP provider in your `terraform.state` file. 
```

Comment:
Nit: Typo in SOPS :)

Reply:
Is there a high-level description we can add here on the method? I suppose you did some careful `terraform state rm ...`?

Reply:
Terraform (at least v0.13.4) prompts you do run this command when you a change the provider
```bash
terraform state replace-provider registry.terraform.io/-/sops registry.terraform.io/carlpett/sops
```
FYI

Reply:
```suggestion
A prerequisite when converting is that you must remove the data source block from the previous SOP provider in your `terraform.state` file. 
```
Extra 'from' in here :)